### ISSUE TITLE
Rails generate rspec:swagger creating spec file only in request folder(not creating initializer folder) + output not matches json format

### ISSUE THREAD

**Issue by taeyeob** [rswag/rswag#505]
Ruby version: 2.5.3
Rails version: 5.2.1

Since this is my first time using swagger in my project, I have searched through the internet several times and had read through the document.
I've recognized that initialized folder is created when generating rspec swagger, but the result I've got is spec.rb file created in requests folder(also newly created) to spec. This is my input and output:
`rails generate rspec:swagger API::V2::MyController
[DEPRECATED] `Bundler.with_clean_env` has been deprecated in favor of `Bundler.with_unbundled_env`. If you instead want the environment before bundler was originally loaded, use `Bundler.with_original_env` (called at /Users/vb/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/spring-2.0.2/lib/spring/application_manager.rb:95)
Running via Spring preloader in process 2100
      create  spec/requests/api/v2/my_spec.rb`

And  of course, I've altered MyController to my controller name. After getting this result, since it is described that the gem searches for initializations in spec/requests, I thought it will make no problem, but it made me to add parameters each time I try to upload at swagger, which conflicts to my intention on trying to use Rswag.

In conclusion, I wonder if there are any way to create mycontroller_spec file to integration folder, as described in this repository.

Thank you for expending your time to read this, and hopefully it will be figured out.
